lib/services/nodes-api-service.js

```diff
@@ -509,6 +509,26 @@ function nodeApiServiceFactory(
     };
 
     /**
+     * Remove the tag from a list of nodes
+     * @param  {String}     tagName
+     * @return {Promise}
+     */
+    NodeApiService.prototype.masterDelTagById = function(tagName) {
```

Comment:
The term Master is used to imply the fact that its a "master delete" of the specified tag from all nodes that have it. This is an add-on to already existing "delete a tag from a specific node",

Agree that it is by tagName and not Id.